web/src/screens/filters/sections/Actions.tsx

```diff
@@ -134,44 +134,67 @@ const TypeForm = (props: ClientActionProps) => {
     }
 
     setPrevActionType(action.type);
-  }, [action.type, idx, setFieldValue]);
+  }, [action.type, idx, prevActionType, setFieldValue]);
 
   switch (action.type) {
   // torrent clients
-  case "QBITTORRENT":
+  case "QBITTORRENT": {
     return <FilterActions.QBittorrent {...props} />;
-  case "DELUGE_V1":
-  case "DELUGE_V2":
+  }
+  case "DELUGE_V1": {
+    return <FilterActions.Deluge {...props} />;
+  }
+  case "DELUGE_V2": {
     return <FilterActions.Deluge {...props} />;
-  case "RTORRENT":
+  }
+  case "RTORRENT": {
     return <FilterActions.RTorrent {...props} />;
-  case "TRANSMISSION":
+  }
+  case "TRANSMISSION": {
     return <FilterActions.Transmission {...props} />;
-  case "PORLA":
+  }
+  case "PORLA": {
     return <FilterActions.Porla {...props} />;
+  }
   // arrs
-  case "RADARR":
-  case "SONARR":
-  case "LIDARR":
-  case "WHISPARR":
-  case "READARR":
```

Comment:
Does ESLint complain about these? IMO the old style is better for this and other places doing the same type of multi case matching.

Reply:
If you leave it as it was, it does not complain.
If you set the curly braces it complains with:
`TS7029: Fallthrough case in switch.` and
`ESLint: Expected a 'break' statement before 'case'.(no-fallthrough)`

Reply:
Then I'd suggest to revert these back to the fall-through style.